README.md

```diff
@@ -1 +1,17 @@
-# -Hello-Microverse-
\ No newline at end of file
+# Hello-Microverse
```

Comment:
- [ ] [OPTIONAL] You should use a descriptive description on your PR, kindly change your description so it reflects the changes on the Pull Request